src/js/components/modals/ServiceFormModal.js

```diff
@@ -373,16 +375,38 @@ class ServiceFormModal extends mixin(StoreMixin) {
     let {jsonDefinition, jsonMode, service} = this.state;
 
     if (jsonMode) {
+      let toolTipContent = (
+        <div>
+          Use the JSON editor to enter Marathon Application definitions manually.
+          {' '}
+          <a href={`${Config.marathonDocsURI}generated/api.html#v2_apps_post`} target="_blank">
+            Read more here.
```

Comment:
Maybe this should say See API docs or See API documentation.
What do you think?